configure.ac

```diff
@@ -320,45 +320,6 @@ fi
 # features that Autoconf doesn't tell us about
 #
 
-
-#
```

Comment:
Should we perhaps keep this check in autoconf -- but rather than provide two implementations, just cause a failure?